decidim-core/app/services/decidim/resource_search.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+module Decidim
+  # This is the base class to be used by other search services.
+  # Searchlight documentation: https://github.com/nathanl/searchlight
+  class ResourceSearch < Searchlight::Search
+    # Initialize the Searchlight::Search base class with the options provided.
+    #
+    # scope - The scope used to create the base query
+    # options - A hash of options to modify the search. These options will be
```

Comment:
Can you document the options not used by searchlight? Like the paginate ones and other if there are any